### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by howardchung**
* Move some strings out into language file
* Reduce amount of footer text
* Handle undefined fields for unparsed matches

### CODE REVIEW

**Comment by howardchung** [comment]
Reviewers:
@blukai 
@mike-robertson 
@nicholashh 
@albertcui (maybe)

File: src/components/Footer/Links.jsx [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
           className={styles.iconButton}
-          tooltipPosition="top-center"
-          tooltip="Steam"
           href="//steampowered.com"
         >
           <IconSteam style={{ width: 14, height: 14 }} />
         </IconButton>
-        Cheese icon by <a href="//www.flaticon.com/authors/belc" target="_blank" rel="noopener noreferrer">Belc </a>
-        on <a href="//www.flaticon.com" target="_blank" rel="noopener noreferrer">Flaticon</a>
+        {strings.cheese_credit}<a href="//www.flaticon.com/authors/belc" target="_blank" rel="noopener noreferrer"> Belc on Flaticon</a>
```
Comment (howardchung) [inline src/components/Footer/Links.jsx]: on should be replaced by a non-language character like '/' or '@'

Comment (albertcui) [inline src/components/Footer/Links.jsx]: @

**Comment by howardchung** [comment]
@nicholashh , it seems like @blukai is making a new cheese icon, so the attribution thing is moot.  I'll revert your  changes and merge this in.

**Pull request merged**